### ISSUE TITLE
target pattern parsing errors are too wordy

### ISSUE THREAD

**Issue by benjaminp** [bazelbuild/bazel#5492]
```
$ bazel version
Build label: 0.15.0- (@non-git)
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Tue Jun 26 15:05:26 2018 (1530025526)
Build timestamp: 1530025526
Build timestamp as int: 1530025526
$ bazel build :
ERROR: Skipping ':': invalid target format ':': invalid target name '': empty target name
WARNING: Target pattern parsing failed.
ERROR: invalid target format ':': invalid target name '': empty target name
INFO: Elapsed time: 0.057s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
```
1. One `ERROR` line is probably enough. We would probably be okay without the `WARNING`, too.
2. Not all error messages are improved by concatenating them into a chain with colons.

**Comment by ulfjack** [comment]
Yeah, I also noticed that. Very annoying.

The colons are due to exception chaining inside Bazel, I think.

**Comment by benjaminp** [comment]
@bazelbuild/triage not stale

**Comment by benjaminp** [comment]
```
$ bazel version
Build label: 7.2.1
Build target: @@//src/main/java/com/google/devtools/build/lib/bazel:BazelServer
Build time: Tue Jun 25 15:53:05 2024 (1719330785)
Build timestamp: 1719330785
Build timestamp as int: 1719330785
l$ bazel build :
WARNING: Target pattern parsing failed.
ERROR: Skipping ':': invalid target name '': empty target name
ERROR: invalid target name '': empty target name
INFO: Elapsed time: 0.147s
INFO: 0 processes.
ERROR: Build did NOT complete successfully
```
